urlhaus.go

```diff
@@ -70,6 +88,65 @@ func GetAllOnlineURLs(ctx context.Context) ([]URLEntry, error) {
 	return ret, nil
 }
 
+// SubmitURLs takes a list of URLs and attempsts to submit them. The list returned wil contain the URLs that were successfully submitted
+func SubmitURLs(ctx context.Context, urls []string, apiKey string, tags []string, threat string) (string, error) {
+	submission := &Submission{}
+	submission.Token = apiKey
+	submission.Anonymous = "0"
+	submission.Submission = []submissionURL{}
+
+	for _, url := range urls {
+		if url == "" {
+			continue
+		}
+
+		urlEntry := &submissionURL{}
+		urlEntry.URL = url
+		urlEntry.Tags = tags
+		urlEntry.Threat = threat
+		submission.Submission = append(submission.Submission, *urlEntry)
+	}
```

Comment:
Instead of doing this processing, why not just let the user submit a `Submission`?  Then they could specify tags per `submissionURL`.  Is it the case that the tags will usually be the same?

Reply:
yeah the idea is you submit in bulk for each set of URLs, figured it was easier but i can switch it if you want